src/App.js

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
 import Calculator from './components/Calculator';
+import NavBar from './components/NavBar';
+import Home from './pages/Home';
+import Quotes from './pages/Quotes';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
#### OPTIONAL 

- [ ] Kindly avoid disabling linters as it is not a good practice, it is better for your linters to pass the linter checks 👍🏿

Reply:
@prolajumokeoni Thanks for the feedback, I have just done that